packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,682 @@
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+
+#### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles"
+```
+
+#### Response Types
+
+HTTP 200:
+[Factor Profile Object](#factor-profile-object)
+
+### Delete a Factor Profile
+
+<ApiOperation method="delete" url="/api/v1/org/factors/${factorName}/profiles/${profileId}" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+| `profileId`  | String | The factor profile's unique identifier.         |
+
+#### Request Example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}"
+```
+
+#### Response Types
+
+HTTP 204:
+*No Content*
+
+### Update a Factor Profile
+
+<ApiOperation method="put" url="/api/v1/org/factors/${factorName}/profiles/${profileId}" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+| `profileId`  | String | The factor profile's unique identifier.         |
+
+#### Request Example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "PIN code",
+  "default": false,
+  "settings": {}
+}' "https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}"
+```
+
+#### Response Types
+
+HTTP 200:
+[Factor Profile Object](#factor-profile-object)
+
+### Create a Factor Profile
+
+<ApiOperation method="post" url="/api/v1/org/factors/${factorName}/profiles" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+
+#### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "PIN code",
+  "default": false,
+  "settings": {}
+}' "https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles"
+```
+
+#### Response Types
+
+HTTP 204:
```

Comment:
Isnt it `201` for created response

Reply:
Do we also want to say that factorProfile name should be unique for a given factorName. You cant have 2 factorProfiles with the same name for a given `factorName`.
Code validation checks that.
cc @jakubvul

Reply:
Sounds like an important point.

Reply:
Confirmed its 200